### ISSUE TITLE
Feature Request: Additional macro options.

### ISSUE THREAD

**Issue by gotex007** [rljohn/pibuddy#2]
Is it possible to have some additional macro options like the nochanneling conditional and having the option to remove the @player part?

**Comment by rljohn** [comment]
The macro is pushing against the 255 macro character limit for cross realm names, so adding nochanneling conditionals to each cast line wouldn't fit unless we find a way to shorten/remove other lines.

**Comment by gotex007** [comment]
Yeah I did notice that. I'd personally be unticking some of the options (like the @player line) to make room for the nochanneling conditional. I tried doing it myself but the addon just overwrites any changes to the macro made manually when I select a buddy since it has to generate the macro again.

**Comment by gotex007** [comment]
There are also some small things that can be done to reduce macro character count. /use can be used instead of /cast (very minor). The server name also isn't required to cast the spell on them and I'd imagine it's very unlikely you would ever be in a group with two people of literally the exact name. Another thing that I don't think would run into issues unless the target is mind-controlled is being able to take out the "help" conditional in the casting at your pibuddy line and the /stopmacro line since they should always be an ally when you're trying to cast it. These three things would save like 25-49 characters, depending on the length of the server name.

**Comment by gotex007** [comment]
Can also use harm instead of nohelp to save 2 characters.

**Comment by gotex007** [comment]
PIing yourself while having the Twins of the Sun Priestess talent still PIs a random target as well so you actually end up not needing the optional "PI a random ally" part of the macro ever.
